core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -446,7 +446,7 @@ private[spark] object UIUtils extends Logging {
       val xml = XML.loadString(s"""<span class="description-input">$desc</span>""")
 
       // Verify that this has only anchors and span (we are wrapping in span)
-      val allowedNodeLabels = Set("a", "span")
+      val allowedNodeLabels = Set("a", "span", "br")
```

Comment:
@tdas Is this change okay? Need it to add line breaks in the job description cells.